src/chains.rs

```diff
@@ -242,21 +243,29 @@ impl ChainItem {
         context: &RewriteContext<'_>,
         shape: Shape,
     ) -> Option<String> {
+        let callee_str = rewrite_ident(context, method_name);
         let type_str = if types.is_empty() {
             String::new()
         } else {
-            let type_list = types
-                .iter()
-                .map(|ty| ty.rewrite(context, shape))
-                .collect::<Option<Vec<_>>>()?;
-
-            format!("::<{}>", type_list.join(", "))
+            // subtract `{calle_str}::<>()` from the shape. We can only place everything on a single
```

Comment:
Random passerby here, saw a typo:
```suggestion
            // subtract `{callee_str}::<>()` from the shape. We can only place everything on a single
```

Reply:
I appreciate you calling it out!

Reply:
I appreciate you working on a tool I literally use hundreds of times a day!!

Reply:
Thank you so much! It honestly means a lot to hear that ❤️